File: Qa_Automation/Selenium_C#/blazingChat/VisualRegression.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using Applitools;
using Applitools.Selenium;
using NUnit.Framework;
namespace blazingChat
{
    [TestFixture]
    public class VisualRegression
    {
        [SetUp]
        public void Setup()
        {
            driver = new ChromeDriver();
            eyes = new Eyes();
            eyes.ApiKey = "YOUR_APPLITOOLS_API_KEY";
        }

        [Test]
        public void Test1()
        {
            Assert.Pass();
        }
    }
}

File: Qa_Automation/Selenium_Java/blazingchat/src/test/java/com/blazinchat/VisualRegression.java
package com.blazinchat;

import com.applitools.eyes.selenium.Eyes;
import com.applitools.eyes.RectangleSize;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.annotations.AfterMethod;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

public class VisualRegression {
    private WebDriver driver;
    private Eyes eyes;

    @BeforeMethod
    public void setUp() {
        // Set the path to the ChromeDriver executable
        System.setProperty("webdriver.chrome.driver", "chromedriver.exe");

        // Create a new ChromeDriver instance
        driver = new ChromeDriver();

        // Initialize the Eyes SDK
        eyes = new Eyes();
        eyes.setApiKey("zOjE51dXy6haYt31xaI2e3KETIvIP6pnSOXlLez106sHM110");
    }

    @Test
    public void testVisualRegression() {
        // Open the website
        driver.get("https://www.warhammer-community.com");

        // Start the visual regression test
        eyes.open(driver, "Warhammer Community", "Home Page", new RectangleSize(800, 600));

        // Perform visual checks on the page
        eyes.checkWindow("Home Page");

        // Close the visual regression test
        eyes.closeAsync();
    }

    @AfterMethod
    public void tearDown() {
        // Close the browser
        driver.quit();

        // Abort the Eyes test if it's still running
        eyes.abortIfNotClosed();
    }
    
}
